pontoon/batch/utils.py

```diff
@@ -60,19 +96,26 @@ def find_and_replace(translations, find, replace, user):
     now = timezone.now()
     translations_to_create = []
     for translation in translations:
+        string = translation.string
```

Comment:
I'd add a comment here explaining why it's helpful to cache the old value of the string.